src/css/_color-palette.react.css

```diff
@@ -9,14 +9,14 @@
     flex-direction: row;
 }
 
-.palette-color {
+.colorpalette .colorpalette-color {
```

Comment:
Once you namespace the class name, it's less important to have to nest the class in this way - we should just use `.colorpalette-color` as the selector.